test/e2e/systemd_test.go

```diff
@@ -113,12 +113,12 @@ WantedBy=default.target
 		cgroupPath := podmanTest.Podman([]string{"inspect", "--format='{{.State.CgroupPath}}'", ctrName})
 		cgroupPath.WaitWithDefaultTimeout()
 		Expect(cgroupPath).Should(Exit(0))
-		Expect(result.OutputToString()).To(Not(ContainSubstring("init.scope")))
+		Expect(cgroupPath.OutputToString()).To(Not(ContainSubstring("init.scope")))
```

Comment:
EXTRA ATTENTION HERE, PLEASE! I'm pretty sure this was an existing bug (using `result`), but please give extra scrutiny here in case there could be unintended consequences.